pages/listings/create/index.html

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <link rel="stylesheet" href="/dist/css/index.css" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="create job listings" />
     <title>Noroff Job Create Listing</title>
   </head>
   <body class="d-flex flex-column flex-fill min-vh-100 gap-4" id="homepage">
```

Comment:
Maybe change the body Id to match what the page is targeted to ?

Reply:
Good catch!